src/app/pages/mini-games/find-pokemon-name/game/components/RoundResult.jsx

```diff
@@ -0,0 +1,35 @@
+import { useCallback, useContext, useEffect } from 'react';
+import { FindPokemonNameContext } from '../state/Context';
+
+const RoundResult = () => {
+  const { state, startGame } = useContext(FindPokemonNameContext);
+  const { gameStatus } = state;
+
+  const onKeyDown = useCallback(
+    (e) => {
+      if (gameStatus !== 'win' && gameStatus !== 'lost') {
+        return;
+      }
```

Comment:
This is **redundant** because `RoundSelect` component is only called with this condition:
```
if (gameStatus === 'win' || gameStatus === 'lost') {
    return <RoundResult />;
}
```

Reply:
Oops I missed it when copy-pasting :D